### PR TITLE
Create appveyor-3.6.yml

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
#906

### CODE REVIEW

File: appveyor-3.6.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  - cmd: set PATH=C:\Ruby27-x64\bin;%PATH%
+  - ps: Bootstrap
+  - git submodule update --init --recursive
+
+environment:
+  USE_RTOOLS: true
+  NOT_CRAN: true
+  KEEP_VIGNETTES: true
+  R_ARCH: x64
+  R_VERSION: 3.6.3
```
Comment (IndrajeetPatil) [inline appveyor-3.6.yml:23]: Testing R3

File: appveyor-3.6.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  - nuget sources add -name funcparser -source https://ci.appveyor.com/nuget/ospsuite-funcparser
+  - nuget sources add -name simmodel -source https://ci.appveyor.com/nuget/ospsuite-simmodel
+  - nuget sources add -name cvodes -source https://ci.appveyor.com/nuget/ospsuite-simmodel-solver-cvodes
+  - nuget sources add -name core -source https://ci.appveyor.com/nuget/ospsuite-core
+  - nuget sources add -name centos -source https://ci.appveyor.com/nuget/ospsuite-centos
+  - nuget install packages.config -OutputDirectory packages # cannot use automatic artifact publish because it's not recognized in a C++/CLI project
+  - rake "prepare_for_build[%APPVEYOR_BUILD_VERSION%]"
+
+build_script:
+  - Rscript -e "install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1-R3/rClr_0.9.1.zip', repos = NULL, type = 'binary')"
```
Comment (IndrajeetPatil) [inline appveyor-3.6.yml:39]: Using R3 specific version of _rClr_

File: appveyor-3.6.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  - path: '*.Rcheck\**\*.out'
+    name: Logs
+
+  - path: '*.Rcheck\**\*.fail'
+    name: Logs
+
+  - path: '*.Rcheck\**\*.Rout'
+    name: Logs
+
+  - path: "ospsuite_*ubuntu18.tar.gz"
```
[the remark this answers is not in the document]
Comment (IndrajeetPatil) [reply appveyor-3.6.yml]: Yes, removed the artifacts section.

**Comment by msevestre** [comment]
I need to merge it on develop for it to be picked uo

**Pull request merged**